scripts/manifests/chromemanifest.json

```diff
@@ -0,0 +1,68 @@
+{
+  "manifest_version": 2,
+  "name": "__MSG_extensionName__",
+  "short_name": "__MSG_extensionNameShort__",
+  "version": "2.0",
+  "author": "rugk, Teal Dulcet",
+
+  "description": "__MSG_extensionDescription__",
+  "homepage_url": "https://github.com/rugk/awesome-emoji-picker",
+
+  "browser_action": {
+    "browser_style": true,
+    "chrome_style": true,
+    "default_icon": "icons/fa-grin-dark.svg",
```

Comment:
Note that Chrome does not support SVGs here.

Reply:
Well… then let'S conver it to a PNG and use that in the manifest for _chrome_ (only).

Reply:
OK, by default this icon is only shown for a fraction of a second before the `IconHandler.js` file runs, so it is only an issue if the user disables the "Colored icon" option. This line would also need to be updated: https://github.com/rugk/awesome-emoji-picker/blob/fa7787aef6b9466f58a0387f0f68ba1b274724be/src/common/modules/IconHandler.js#L48 but it is currently unreachable.

Reply:
Hmm yeah, but having that option disabled is possible.
ALso yeha, of course it liekyl also needs to load the PNG version there then.

Reply:
Is slmwthonh still needed to be done? So do we have to disable that option?

Reply:
Well, you would need to decide if you want to support these light/dark theme icons in Chrome. If so, your SVGs would need to be converted to a bunch of PNGs, I believe you have Inkscape or some program that can do this easily (see https://github.com/rugk/unicodify/pull/15#discussion_r607853240). If not, I can disable that option in Chrome.

Reply:
Sure no probIem I've committed it and tried to push the PNG versions, but uff:
```
To https://github.com/tdulcet/awesome-emoji-picker
 ! [remote rejected] tdulcet-chrome -> tdulcet-chrome (permission denied)
error: Fehler beim Versenden einiger Referenzen nach 'https://github.com/tdulcet/awesome-emoji-picker'
``` 

Could you please click the button "Allow commits from maintainers" or so at the right side? (Best do this for alll PRs.)
Then I should be able to push it…

Otherwise, here they are:
![fa-grin-light](https://user-images.githubusercontent.com/11966684/181853243-30f7831b-c772-4c4b-b09d-1e88bc717355.png)
![fa-grin-light](https://user-images.githubusercontent.com/11966684/181853244-70918609-76cd-4b8f-aedb-d4173091f298.svg)

Reply:
Thanks. Hmm, that option is already checked, so I am not sure what the issue is:
![image](https://user-images.githubusercontent.com/15305150/181892486-b1ece6be-80be-4872-b8c2-db71cd486ee2.png)
Anyway, I just invited you to have full access to my fork, so hopefully that will resolve it.

Reply:
Okay great thanks yeah. Still cannot push it to this branch:
```
$ git push https://github.com/tdulcet/awesome-emoji-picker chrome       
error: src refspec chrome does not match any
error: failed to push some refs to 'https://github.com/tdulcet/awesome-emoji-picker'
```

Same error if I add it:
```
$ git remote add tdulcet https://github.com/tdulcet/awesome-emoji-picker
$ git push tdulcet chrome                                        
error: src refspec chrome does not match any
error: failed to push some refs to 'https://github.com/tdulcet/awesome-emoji-picker'
```

Even that fails:
```
$ git push --set-upstream tdulcet chrome    
error: src refspec chrome does not match any
error: failed to push some refs to 'https://github.com/tdulcet/awesome-emoji-picker'
```'

I could push it to another branch though: https://github.com/tdulcet/awesome-emoji-picker/pull/2

Reply:
Ahhh and uff, maybe that's the reaosn for that error… a lot of unrelated commits… did someone reqrite the history/did a git push --force?

Anyway uhm you may just cherry-pick the latest commit if you want.

ANd possibly consider merging in the main branch in this repo.

Reply:
Ad no, had the wrong target branch. https://github.com/tdulcet/awesome-emoji-picker/pull/2 works now and you can just approve/merge that.

Reply:
Thanks for figuring it out! I merged your PR and will update the Chrome manifest file to use the new PNGs.